apps/builder/app/builder/shared/expression-editor.tsx

```diff
@@ -341,11 +341,9 @@ const emptyScope: Scope = {};
 const emptyAliases: Aliases = new Map();
 
 const wrapperStyle = css({
-  "& .cm-content": {
-    // 1 line is 16px
-    // set and max 20 lines
-    maxHeight: 320,
-  },
+  // 1 line is 16px
+  // set and max 20 lines
+  "--ws-code-editor-max-height": "320px",
```

Comment:
you just violated our RFC https://github.com/webstudio-is/webstudio-rfcs/blob/main/text/002-css-vars.md

"We never copy-paste a CSS variable name across modules."